macros/generate_schema_name.sql

```diff
@@ -1,12 +1,3 @@
 {% macro generate_schema_name(custom_schema_name, node) -%}
-    {%- set default_schema = target.schema -%}
-    {%- if target.name == 'local' -%}
-        {{ default_schema }}
-    {%- else -%}
-        {%- if custom_schema_name is not none -%}
-            {{ custom_schema_name | trim }}
-        {%- else -%}
-            {{ default_schema }}
-        {%- endif -%}
-    {%- endif -%}
+    {{ dbt_common_macros.custom_generate_schema_name(custom_schema_name, node) }}
```

Comment:
Creating a macro that uses our common macros is with the intention to explicitly show the user that this exists? 

When searching for good practice in dbt I found that we can access macro directly from package 
https://docs.getdbt.com/docs/building-a-dbt-project/jinja-macros#using-a-macro-from-a-package

Reply:
generate_schema_name macro is a special kind of macro that overwrote the default one. But you can't overwrite it with macro from a package, it must be done explicitly.
https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas#advanced-custom-schema-configuration